cpp/src/gandiva/tests/date_time_test.cc

```diff
@@ -452,7 +452,8 @@ TEST_F(TestProjector, TestTimestampDiffMonth) {
   std::shared_ptr<Projector> projector;
   auto status =
       Projector::Make(schema, {diff_months_expr}, TestConfiguration(), &projector);
-  std::cout << status.message();
+  // std::cout << status.message();
+  SCOPED_TRACE(status.message());
```

Comment:
Just delete this, no need for scoped_trace here.  If status is not `ok` then the test will fail and it will be obvious where it failed.  Status is not a "test parameter"

Same with the next two occurrences.